src/symbol.rs

```diff
@@ -123,35 +123,6 @@ pub(super) fn add_executable_section_symbols(
     syms
 }
 
-pub(super) fn append_dummy_symbol(mut syms: Symbols, name: &str) -> Symbols {
```

Comment:
The failures in the tests are the result of the removal of this function. We emit a dummy `PUBLIC` symbol after the very last symbol we have as a catch-all to pointers that fall within the executable but not within the last symbol. To fix this you could modify the code in `add_executable_section_symbols()` to add the dummy symbol when encountering PE files (it only handles ELF now).

Reply:
Thanks, I've patched this in now. All tests pass now, except for `mozwer`.